config/initializers/filter_parameter_logging.rb

```diff
@@ -3,39 +3,36 @@
 # Configure parameters to be filtered from the log file. Use this to limit dissemination of
 # sensitive information. See the ActiveSupport::ParameterFilter documentation for supported
 # notations and behaviors.
-Rails.application.config.filter_parameters += %i[
-  _key
-  address
-  address_line_one
-  address_line_two
-  application_start_date
-  certificate
-  client_id
-  client_secret
-  code
-  country
-  crypt
-  date_of_birth
-  email
-  first_name
-  hearing_court_name
-  hearing_date
-  last_name
-  name
-  nino
-  oid
-  other_names
-  otp
-  passw
-  pkce
-  postcode
-  reason
-  salt
-  secret
-  ssn
-  submission_date
-  telephone_number
-  tenant_id
-  token
-  urn
+Rails.application.config.filter_parameters += [
+  :_key, :certificate, :client_id, :client_secret,
+  :oid, :otp, :passw, :pkce, :salt, :secret, :ssn,
+  :tenant_id, :token,
+
+  # Attributes relating to an application
+  # It does partial matching (i.e. `telephone_number` is covered by `phone`)
+  :address,
+  :address_line_one,
+  :address_line_two,
```

Comment:
Wondering if we want to align this with Apply, where it is :address_line - and will pick up :address_line_one, and :address_line_two with partial matching. :address is already picking them up, but we want that to be tighter on that as it will also be picking up :email_address etc.

Reply:
Thanks will amend